.github/workflows/deploy-daily-snapshot.yml

```diff
@@ -34,4 +36,5 @@ jobs:
           slack_token: ${{ secrets.SLACK_TOKEN }}
           working_directory: terraform/daily_snapshot
           environment: Snapshot Service
-          nr_license_key: ${{ secrets.NR_LICENSE_KEY }}
+          new_relic_account_id: ${{ secrets.NEW_RELIC_ACCOUNT_ID }}
```

Comment:
I don't quite get it. Here you assign `NEW_RELIC_ACCOUNT_ID` and `NEW_RELIC_API_KEY` and in [README](https://github.com/ChainSafe/forest-iac/blob/samuel/workflow-path-vr/terraform/daily_snapshot/README.md#required-environment-variables) there is only the license key. What is the "correct" way of setting it all up?

Reply:
Let's just not forget to update the README. Outdated documentation is fairly frustrating.